### PR TITLE
Add `php-cs-fixer` to the default set of tools, when configuration for it is detected

### PR DESCRIPTION
**Pull request by basz**
Signed-off-by: Bas Kamer <...@bushbaby.nl>

|    Q          |   A
|-------------- | ------
| New Feature   | yes

### Description

This adds direct support for PHP CS Fixer avoiding additional_checks foo in .laminas-ci.json

### CODE REVIEW

**Review by Ocramius — CHANGES_REQUESTED** [review]
Looking ok, but a test addition is needed 👍

**Comment by basz** [comment]
there you go...

File: src/tools.ts [hunk 1]
```diff
@@ -115,6 +115,13 @@ export default function createTools(config: Config): Array<Tool> {
             command       : './vendor/bin/codecept run',
             filesToCheck  : [ 'codeception.yml.dist', 'codeception.yml' ],
             toolType      : ToolType.CODE_CHECK,
+        },
+        {
+            executionType : ToolExecutionType.STATIC,
+            name          : "PHP CS Fixer",
```
Comment (Ocramius) [inline src/tools.ts]: Could you adjust the warnings too?

**Review by Ocramius — APPROVED** [review]
Thanks @basz!

**Comment by boesing** [comment]
Isnt there also a dist file? Should be added to the files as well imo.

**Pull request merged**